Waffle octobre fix

Thinking-waffle opened the pull request:
This follows and complete the previous one, the details can be annoying while avoiding github desktop.

Wait did I change the name of the branch instead?

Did I create one or two pull requests? Oh Well just be cautious while merging all of this but I see that everything I have changed seems to be here.

Thinking-waffle committed:
sami characters

Thinking-waffle committed:
sami history file for sami characters

Thinking-waffle committed:
plane retexture files

Iceland Quebec Finland Sami

Thinking-waffle committed:
plane textures

Thinking-waffle committed:
localization for new trait

Thinking-waffle committed:
new trait for a japanese aide de camp

Thinking-waffle committed:
modified JAP_shigeru_honjo

Thinking-waffle committed:
effect related with JAP_shigeru_honjo

Thinking-waffle committed:
kodoha militarists have enough war support

Thinking-waffle committed:
extra advisor is not free

Thinking-waffle committed:
Fix sami advisors and added one

Thinking-waffle committed:
removed the wrong file and fix the right thing

Thinking-waffle committed:
Update r56_traits.txt

militaristic -> jingoistic

Thinking-waffle committed:
update jingoistic trait

Thinking-waffle committed:
Update JAP_l_english.yml

Thinking-waffle committed:
Update loading_tips_l_english.yml

Finnish quote

Thinking-waffle committed:
A port for Viipuri

Got presented convincing evidence so I added a level 1 port, could maybe be level 2.

Thinking-waffle committed:
Added compliance and hopefully fixed Jordanian integration by Saudis

Thinking-waffle committed:
showa statist trait for kodoha enthusiast.

Thinking-waffle committed:
JAP_national_socialst_showa_statist

Thinking-waffle committed:
Update r56_traits.txt

Thinking-waffle committed:
Added Seigo Nakano

Thinking-waffle committed:
Recruit Seigo Nakano

Thinking-waffle committed:
adding tooltip for new advisor in challenge the Zaibatsus

Thinking-waffle committed:
purging kodoha will retire Sadeo Araki

He could be recalled as totalitarian minister of education to counter balance the control faction,

Thinking-waffle committed:
Made Sadeo Araki more relevant for Kodoha campaigns

Thinking-waffle committed:
added Kodoha chief Jinzaburo Masaki

Thinking-waffle committed:
Recruit Jinzaburo Masaki

Thinking-waffle committed:
Expanded kodoha faction

Thinking-waffle committed:
Update japan.txt

Thinking-waffle committed:
fixed a typo

marijn211 committed:
Fixed effect/condition scope mixup

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-octobre-fix

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-octobre-fix

marijn211 committed:
Fix plane errors (I hope)

marijn211 committed:
Update SAU.txt

marijn211 reviewed (changes requested):
`[17:33:14][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7224
[17:33:14][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7236
[17:33:14][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7248
[17:33:14][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7260
[17:33:14][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7272
[17:33:14][no_game_date][pdx_entity.cpp:2115]: Couldn't find mesh "FIN_cas_1_plz_p23_mesh" (or FIN_cas_1_plz_p23_mesh is not of 'pdxmesh' type) [FIN_CAS_equipment_entity]
[17:33:14][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity FIN_medium_plane_entity, in file "gfx/entities/units_planes.asset", near line 10343
[17:33:14][no_game_date][pdx_entity.cpp:2064]: Duplicate of FIN_light_plane_0_entity added to entity system
[17:33:14][no_game_date][pdx_entity.cpp:2064]: Duplicate of ICE_light_plane_0_entity added to entity system
[17:33:14][no_game_date][pdx_entity.cpp:2064]: Duplicate of ICE_CAS_equipment_entity added to entity system
[17:33:14][no_game_date][pdx_entity.cpp:2192]: Duplicate of ICE_CAS_equipment_entity added to entity system
[17:33:14][no_game_date][pdx_entity.cpp:2064]: Duplicate of ICE_jet_plane_entity added to entity system
[17:33:14][no_game_date][pdx_entity.cpp:2064]: Duplicate of ICE_light_plane_entity added to entity system
[17:33:14][no_game_date][pdx_entity.cpp:2064]: Duplicate of ICE_medium_plane_entity added to entity system`

Thinking-waffle committed:
Update r56_meshes_planes.gfx

Thinking-waffle committed:
Update r56_meshes_planes.gfx

Thinking-waffle committed:
missing finnish cas texture

Thinking-waffle committed:
Update units_planes.asset

Should fix planes because the new Finnish planes weren't at the top and therefore couldn't be cloned

Thinking-waffle committed:
Update units_planes.asset

Removing duplicates

Thinking-waffle committed:
Update 01_bba_plane_icons.txt

should do nothing or help fix that weird missing 3d model bug reported the other day

marijn211 commented:
```
[20:40:06][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity FIN_medium_plane_entity, in file "gfx/entities/units_planes.asset", near line 2856
[20:40:06][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7515
[20:40:06][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7527
[20:40:06][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7539
[20:40:06][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7551
[20:40:06][no_game_date][assetfactory.cpp:877]: Couldn't find parent clone entity CHI_CAS_equipment_0_entity, in file "gfx/entities/units_planes.asset", near line 7563
```

Thinking-waffle committed:
Update R56_news_l_english.yml

rewrote the Napoleon VI rise to power news event

Thinking-waffle committed:
Update R56_news_l_english.yml

Thinking-waffle committed:
Fixing unit_planes.asset

Turns out a Finnish model was still cloned before its actual model.

The Chinese bug was caused by a an inversion of two letters in "equpiment" while the warlord version was correct

Thinking-waffle committed:
Update _generic_decisions.txt

Added decisions to buy and refit ships

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-octobre-fix

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-octobre-fix

marijn211 committed:
Undo introduction of duplicate decision
